src/PhpPact/Consumer/TestCase/MockServerTestCase.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace PhpPact\Consumer\TestCase;
+
+use PhpPact\Http\GuzzleClient;
+use PhpPact\Standalone\MockServer\MockServer;
+use PhpPact\Standalone\MockServer\MockServerConfig;
+use PhpPact\Standalone\MockServer\MockServerEnvConfig;
+use PhpPact\Standalone\MockServer\Service\MockServerHttpService;
+use PHPUnit\Framework\TestCase;
+
+class MockServerTestCase extends TestCase
+{
+    /** @var MockServerConfig */
+    protected $config;
+    /** @var MockServer */
+    private $mockServer;
+
+    protected function setUp()
+    {
+        $this->config = new MockServerEnvConfig();
+        $this->config
+            ->setPactFileWriteMode('merge')
+            ->setPactDir('D:/Development/PACT/pact-php');
```

Comment:
Can this be a relative path?

Reply:
Yes, you can definitely use a relative path. This file should have never been committed. It was just something I was experimenting with.